crates/autopilot/src/arguments.rs

```diff
@@ -392,6 +399,7 @@ impl FromStr for FeePolicyKind {
                     .ok_or("missing surplus factor")?
                     .parse::<f64>()
                     .map_err(|e| format!("invalid surplus factor: {}", e))?;
+                validate_factor(factor)?;
                 let max_volume_factor = parts
```

Comment:
validate also `max_volume_factor` because of this: https://github.com/cowprotocol/services/blob/main/crates/driver/src/domain/competition/solution/scoring.rs#L298

It would fail scoring if we set `max_volume_factor = 1.0` (for example, if we want to make sure no capping is done)